specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/azureFirewall.json

```diff
@@ -688,6 +688,10 @@
         "additionalProperties": {
           "$ref": "#/definitions/AzureFirewallAdditionalProperties",
```

Comment:
>manually [](http://example.com/codeflow?start=40&length=8)

the name of the property suggests it is not used for manual scaling

Reply:
the customer is manually providing the autoscaleConfiguration